### PR TITLE
 #162263189 Define endpoints for redflags

### PR DESCRIPTION
**Pull request by Curti-s**
### What does this PR do?
It ensures that tests are created for the CRUD operations on red flags

### Description of the task completed
Define test for GET, PUT, POST and DELETE

### How can this be manually tested?
Run pytest

### Screenshot
screenshot from 2018-11-29 01-29-08
screenshot from 2018-11-29 01-29-13
screenshot from 2018-11-29 01-29-15

### Relevant pivotal tracker stories
#162263189

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 venv/
-__pycache__/
\ No newline at end of file
+__pycache__/
```
Comment (thoth-ky) [inline .gitignore:2]: Add `.coverage` to gitignore

File: .travis.yml [hunk 2]
```diff
@@ -4,5 +4,14 @@ python:
 # command to install dependencies
 install:
   - pip install -r requirements.txt
+  - pip install python-coveralls
```
Comment (thoth-ky) [inline .travis.yml:7]: Add these to the requirements file so you do not have to `pip install ` each

File: app/api/v1/views/redflag_view.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        self.db = redflag_list
+
+    # get a red flag
+    def get(self,id):
+        response = self.db[id]
+        if len(response) < 0:
+            abort(404)
+        return make_response(jsonify(response),200)
+
+    # # edit a red flag comment / locaion
```
Comment (thoth-ky) [inline app/api/v1/views/redflag_view.py:63]: Remove any commented code

File: tests/v1/test_redflag.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        self.assertEqual(response.status_code, 201)
+        # get response data in json format
+        results = json.loads(response.data.decode())
+        result = self.client().get(
+                '/redflag/{}/'.format(results['data'][0]['id']))
+        # assert that redflag is returned given its actual id
+        self.assertEqual(result.status_code, 200)
+ 
+
+    # def test_edit_a_redflag(self):
```
Comment (thoth-ky) [inline tests/v1/test_redflag.py:64]: Remove commented code

**Review by thoth-ky — CHANGES_REQUESTED** [review]
Kindly check out my suggestions and implement the feedback.

**Comment by thoth-ky** [comment]
your PR title doesn't exactly match what you say it does. Try separate concerns and have different PRs for different concerns

**Comment by Curti-s** [comment]
Will do so

**Pull request merged**